### DISCUSSION TITLE
curve tab Incorrect display

### DISCUSSION THREAD

**Discussion by 4SM** [black7375/Firefox-UI-Fix#698] [Q&A]
![Screenshot - 2023-05-07 , 22_59_09](https://user-images.githubusercontent.com/13938189/236685485-2a725dab-66c6-4dcc-965c-cef29cfcf5c2.png)

**Comment by black7375** [comment]
This bug can be reproduced depending on the resolution or OS, and it is a little tricky.
See #629
